langchain/src/memory/motorhead_memory.ts

```diff
@@ -19,13 +19,18 @@ export interface MotorheadMemoryMessage {
 export type MotorheadMemoryInput = BaseChatMemoryInput &
   AsyncCallerParams & {
     sessionId: string;
-    motorheadURL?: string;
+    url?: string;
     memoryKey?: string;
     timeout?: number;
+    apiKey?: string;
+    clientId?: string;
   };
 
+const MANAGED_URL = "https://api.getmetal.io/v1/motorhead";
+// const LOCAL_URL = "localhost:8080"
+
 export class MotorheadMemory extends BaseChatMemory {
-  motorheadURL = "localhost:8080";
+  url = MANAGED_URL;
```

Comment:
I think this is fair because anyone using this in production probably isn't running it on `localhost`?